lib/utils/fs.go

```diff
@@ -241,32 +241,97 @@ func FSTryReadLockTimeout(ctx context.Context, filePath string, timeout time.Dur
 	return fileLock.Unlock, nil
 }
 
+// RemoveAllSecure is similar to [os.RemoveAll] but leverages [RemoveSecure] to delete files.
```

Comment:
I think we should also mention what makes it secure. Without that context, I might assume that the stdlib is plenty secure as compared to code that we wrote ourselves.

Reply:
Updated the docs, thank you!